### ISSUE TITLE
Possible overpass turbo query

### ISSUE THREAD

**Issue by jschleuss** [socal-osm/review-of-remote-areas#1]
cc @kingrollo 

http://overpass-turbo.eu/s/h9Q

```
/*
This has been generated by the overpass-turbo wizard.
The original search was:
“highway=track and highway=residential”
*/
[out:json][timeout:25];
// gather results
(
  // query part for: “highway=track and highway=residential”
  way["tiger:reviewed"=no]["highway"~"track|residential"]({{bbox}});
);
// print results
out body;
>;
out skel qt;
```

Maybe we could use that and just delete the "tiger:reviewed=no" tag, which is pretty visible in JOSM. We could make this a big MapRoulette project.

I also think we could use the county's aerial imagery, which I believe is the USGS large 2014 set with road labels (helpful in naming).

**Comment by techlady** [comment]
Why delete that particular tag (just so I can understand)?

**Comment by jschleuss** [comment]
@techlady in JOSM those tiger:reviewed=no get a kind of baby poop green highlight. I guess I just equated that with bad. But we could swap it to yes. Here's [some discussion](http://wiki.openstreetmap.org/wiki/TIGER_to_OSM_Attribute_Map#Misc_Attribute_Pairs#Misc_Attribute_Pairs) about this tag.

> tiger:reviewed=no was set on all ways in the TIGER import. It is intended as a way of tracking TIGER fixup progress. However it does not succeed in this aim and is largely pointless.
> Some people remove this tag to indicate that they have reviewed (and perhaps corrected) the imported TIGER data. Some people set the value to yes. Some people set the value to 'aerial', indicating that they used aerial imagery to check/fix the location.
> There is some support within JOSM and Potlatch for differing displays according to this tag.
> However TIGER fixup is quite laborious enough without having to fiddle with this tag, so many people don't bother doing anything, in order to make faster progress with the fixup itself. Others object to the tag since it implies a process of review and "moderation" of edits, which does not exist (and we're proud that it doesn't exist). Some people don't modify the tag because they simply don't know about it. All this means that the tag is not an accurate indication of anything, and developers of tools such as TIGER Edited Map have developed different more reliable ways of seeing areas which need fixup.

There is this [ITO map](http://product.itoworld.com/map/162?lon=-118.19834&lat=34.46430&zoom=12) as another alternative.

**Comment by techlady** [comment]
Yes, I saw the discussion. So it falls into the category of "a little annoyance." But a bot fix could produce unwanted results. One bot was designed to fix all the abbreviations on roads, such as Cr. becoming "Creek" or "Circle." There are many roads in the Four Corners states whose names now are something like "Creek 24," because it used to be CR 24 (County Road 24). I still find those. Obviously, this change is different. But in Maproulette a real person looks at each one.

**Comment by techlady** [comment]
Come to think of it, fixing the "Creeks" would be a good Maproulette task.

**Comment by jschleuss** [comment]
ha! Yeah. No bots for this one. I think the best thing would be to just create the Maproulette task and farm it out to humans. I recently took a stab at a lot of these near Agua Dulce and I think that's the best approach. I'll create a task and then share it back you @techlady and @kingrollo

**Comment by jschleuss** [comment]
Here's a tester http://maproulette.org:8080/map/237

**Comment by michaelkirk** [comment]
So, for example, if that line matches up with the satellite imagery, all that needs to be done is to remove the tag in (e.g.) iD and marked as fixed in map roulette?

**Comment by jschleuss** [comment]
Correct. Pretty straight-forward and very helpful!

**Comment by kingrollo** [comment]
This looks really good - thanks for setting it up! I'm struggling to see how to use MapRoulette right now, but I like this method...

**Comment by kingrollo** [comment]
OK, ignore that - I think I figured it out. The only problem I can see is that in some locations there are multiple roads or tracks which seem to be represented by a single TIGER line. Take a look [here](http://www.openstreetmap.org/edit#map=19/34.51491/-118.24232). Any thoughts?

**Comment by jschleuss** [comment]
@kingrollo that may be a thing. I'm not totally sure. 
Check out that area with one of these custom tile servers (from L.A. County assessor):
`http://cache.gis.lacounty.gov/cache/rest/services/LACounty_Cache/LACounty_StreetMap/MapServer/tile/{zoom}/{y}/{x}`
`http://cache.gis.lacounty.gov/cache/rest/services/LACounty_Cache/LACounty_Base/MapServer/tile/{zoom}/{y}/{x}`

**Comment by kingrollo** [comment]
Interesting. So should we be talking about somehow comparing against the LaCo data as well?

**Comment by jschleuss** [comment]
It might be a good idea. The county regularly updates a [file of addresses and streets](http://egis3.lacounty.gov/dataportal/2012/06/19/la-county-address-points/) that we'll be using in an upcoming address import. It basically shows what you see in those assessor tiles. I think it would be good to add those streets, but am cautious about importing them. The current bad data is from a 2005 Census Tiger file uploaded in 2007. If we could find that old tiger file we could do an analysis and find areas where that file diverges with the current county file and then highlight those as possible problem areas.
